consumer-test/integrationTestController.sh

```diff
@@ -57,6 +58,9 @@ TEST_CASES=$(find testCases -name '*.yml')
 while true; do sleep 10; echo "[INFO] Integration tests still running."; done &
 notificationThreadPid=$!
 
+while true; do sleep 40000; echo "[INFO] Here are the preliminary logs:"; ls -laR |grep -i 'log.txt' ;find . -name 'log.txt' |xargs cat; done &
```

Comment:
Just for my curiosity: why not use `find -exec cat` as described here https://unix.stackexchange.com/a/292254? Would this not be the more idiomatic way?

Reply:
All ways leads towards Rome. Changed as you suggested.